packages/modular-scripts/src/__tests__/index.test.ts

```diff
@@ -210,74 +210,219 @@ describe('modular-scripts', () => {
         ),
       ).toMatchInlineSnapshot(`
         Object {
+          "bundledDependencies": Array [],
           "dependencies": Object {
             "react": "17.0.2",
           },
-          "files": Array [
-            "README.md",
-            "dist-cjs",
-            "dist-es",
-            "dist-types",
-          ],
           "license": "UNLICENSED",
-          "main": "dist-cjs/index.js",
           "modular": Object {
             "type": "view",
           },
-          "module": "dist-es/index.js",
+          "module": "static/js/index-IC6FL6E2.js",
           "name": "sample-view",
-          "typings": "dist-types/index.d.ts",
           "version": "1.0.0",
         }
       `);
     });
 
-    it('THEN outputs the correct output cjs file', () => {
```

Comment:
Why are removing the snapshots of the output? This is very useful for visualising bundle output changes when updating the configuration of builds.

Reply:
We're removing the cjs test since now views build only esm modules.